lib/ExportsStorage.js

```diff
@@ -192,6 +203,17 @@ export default class ExportsStorage {
             reject(err);
             return;
           }
+
+          if (!additional) {
+            this.db.run(
+              'INSERT INTO exports_fts VALUES (?, ?, ?, ?)',
```

Comment:
`fts` is for full text search, right? I think it would be okay to spell that out instead of using the abbreviation.